lib/rmt/downloader.rb

```diff
@@ -0,0 +1,137 @@
+module RMT
```

Comment:
That shouldn't be required here as you have the `lib/rmt.rb` file which already does that.
Haven't tried this yet, but I'm quite sure you can just remove `module RMT; end` here.